app/models/gpo_confirmation.rb

```diff
@@ -14,9 +14,6 @@ def entry
   private
 
   def encryptor
-    # This currently uses the SessionEncryptor, which is meant to be used to
-    # encrypt the session. When this code is changed to integrate a new mail
-    # vendor we should create a purpose built encryptor for that vendor
-    Encryption::Encryptors::SessionEncryptor.new
+    Encryption::Encryptors::BackgroundProofingArgEncryptor.new
```

Comment:
maybe add a 3rd encryptor here? since this is for proofing but isn't backgrounded yet

Reply:
I thought about it and didn't feel great about the copy/paste part here. Especially since this does run in a background job when the GPO upload occurs. Maybe a subclass or something like that could work?

Reply:
oh right this is in a daily background job ... nvm let's leave it